### PR TITLE
139456263/Re-design authentication pages

### PR DESCRIPTION
**Pull request by Del-sama**
#### What does this PR do?
* Re-designs UI for user authentication pages.
#### Description of Task to be completed?
* Redesign sign up page
* Redesign sign in page
#### How should this be manually tested?
* Run ```npm start```
* Click on sign up
* Click on sign in
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#139456263 Re-Design UI for user authentication screen
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by iampikuda — APPROVED** [review]

File: public/css/main.css [hunk 1]
```diff
@@ -294,4 +296,729 @@ footer a:hover {
...
+    height: 100%;
+    background-size: 50%;
+    padding-top: 50px;
+  }
+  
+}
+
+/* Smartphones (landscape) ----------- */
+@media only screen and (min-width : 321px) {
+.avatars{
```
Comment (ooduye) [inline public/css/main.css]: Fix indentation

File: public/css/main.css [hunk 2]
```diff
@@ -294,4 +296,729 @@ footer a:hover {
...
+    height: 100%;
+    background-size: 50%;
+    padding-top: 50px;
+  }
+  
+}
+
+/* Smartphones (portrait) ----------- */
+@media only screen and (max-width : 320px) {
+.avatars{
```
Comment (ooduye) [inline public/css/main.css]: Fix indentation

File: public/css/main.css [hunk 3]
```diff
@@ -294,4 +296,729 @@ footer a:hover {
...
+    background-color: #fff;
+    height: 100%;
+    background-size: 50%;
+    padding-top: 50px;
+  }
+  
+}
+
+@media only screen and (min-device-width: 320px) and (max-device-height: 640px) and (orientation : portrait) and (-webkit-device-pixel-ratio: 2){
+.avatars{
```
Comment (ooduye) [inline public/css/main.css]: Repetition of code

**Comment by ooduye** [comment]
When user signs up with existing account, no error message is flagged

**Comment by ooduye** [comment]
When user clicks on sign in from the sign up page, it loads an invalid sign in page. Fix this

**Comment by iampikuda** [comment]
REAPPROVED

**Pull request merged**